website/content/blog/kubecon-na-2025.md

```diff
@@ -0,0 +1,81 @@
+---
+title: KubeCon NA 2025 Schedule Announced!
+author: Evan Anderson, Jennifer Power
```

Comment:
It looks like we need a `date:` header:


```suggestion
date: 2026-08-06 09:00:00 -05:00
author: Evan Anderson, Jennifer Power
```


<img width="1482" height="313" alt="Image" src="https://github.com/user-attachments/assets/5a6ac44a-92b9-4dac-9b19-c9ab0c1e3570" />